context_test.go

```diff
@@ -152,6 +152,14 @@ func (*ContextSuite) TestNewLoggerAddsConfig(c *gc.C) {
 	})
 }
 
+func (*ContextSuite) TestConfigureLoggers(c *gc.C) {
+	context := loggo.NewContext(loggo.INFO)
+	err := context.ConfigureLoggers("testing.module=debug")
+	c.Assert(err, gc.IsNil)
+	expected := "<root>=INFO;testing.module=DEBUG"
+	c.Assert(context.Config().String(), gc.Equals, expected)
```

Comment:
should you be testing that the configured levels actually mean something (eg, debug messages are included for testing.module but not for a different logger?)
I'm slightly worried about user confusion (understanding that each context has its own logger config, etc.) but it does fit the issue we are trying to address (have explicitly different logging config for each model).

Reply:
There are other tests that test that the configured loggers output the right levels, so I don't think it is necessary here.

Yes, I think there is the potential for confusion. The docs are pretty clear though.